src/service/models/user.model.tsx

```diff
@@ -10,20 +11,20 @@ class User {
   id: number;
   isAdmin: boolean;
   lastName: string;
-  organizationId: number;
+  organization: Organization;
   updatedAt: Date;
 
   constructor(props: any) {
     this.id = props.id;
     this.city = props.city;
-    this.contactID = props.contactID;
+    this.contacts = new Contacts(props.contacts);
     this.country = props.country;
     this.email = props.email;
     this.firstName = props.firstName;
     this.functionTitle = props.functionTitle;
     this.isAdmin = props.isAdmin;
     this.lastName = props.lastName;
-    this.organizationId = props.organizationId;
+    this.organization = new Organization(props.organization);
```

Comment:
Can you not just get the organization.id?

Reply:
Hmm, I'm not sure to get your point. When we fetch a user, we also fetch its associated organization (see gql schema)